includes/class.llms.engagements.php

```diff
@@ -637,15 +637,18 @@ private function trigger_engagement( $data, $delay ) {
 	 *
 	 * This is the callback function for deleted engagement posts.
 	 *
+	 * The `deleted_post` action param `$post` has been added since WordPress 5.5.0.
```

Comment:
Good catch. I suggest this just get changed to work on < 5.5.0 only and not worry about whether or not `$post` is passed. We can just always rely on `get_post_type()` and drop the second parameter entirely.

Reply:
we can of course, but if I already have an information ready, why should I call a function?
Why did they pass the `$post` parameter at all? To avoid calling `get_post( $post_id )` I guess, right?
I'd rather add a todo like "remove wp<5.5 compat code when we'll raise minimum wp version to 5.5+".
What do you think?

Reply:
I was just thinking it seems like an bit of logic to worry about unnecessarily. Calling `get_post()` when the post has already been called loads it from memory and won't take that much longer than accessing the post directly. But you're right. Add a note.